packages/hash/frontend/src/components/layout/PageHeader/SearchBar/SearchInput.tsx

```diff
@@ -0,0 +1,181 @@
+import {
+  Box,
+  IconButton,
+  InputAdornment,
+  OutlinedInput,
+  Tooltip,
+  Typography,
+} from "@mui/material";
+import { useCallback, useEffect, useRef } from "react";
+import { useKeys } from "rooks";
+import { faXmark } from "@fortawesome/free-solid-svg-icons";
+
+import { SearchIcon, FontAwesomeIcon } from "../../../icons";
+
+const ClearSearchIcon: React.FC<{
+  clearSearch: () => void;
+}> = ({ clearSearch }) => {
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        height: "100%",
+        alignItems: "center",
+        cursor: "pointer",
+        fontWeight: "bold",
+        width: "26px",
+        justifyContent: "flex-end",
+        marginRight: 1,
+      }}
+    >
+      <Tooltip title="Clear search" placement="right">
+        <IconButton
+          sx={(theme) => ({
+            width: "20px",
+            height: "20px",
+            display: "flex",
+            alignItems: "center",
+            justifyContent: "center",
+            borderRadius: "2px",
+            color: theme.palette.gray[50],
+
+            "&:hover": {
+              transition: theme.transitions.create([
+                "color",
+                "background-color",
+              ]),
+              backgroundColor: theme.palette.gray[20],
+              color: theme.palette.gray[80],
+            },
+          })}
+          onClick={clearSearch}
+        >
+          <FontAwesomeIcon
+            sx={{ width: "12px", height: "12px" }}
+            icon={faXmark}
+          />
+        </IconButton>
+      </Tooltip>
+    </Box>
+  );
+};
+
+const ShortcutIcon = () => {
+  const isMac = navigator.userAgent.toUpperCase().includes("MAC");
+
+  return (
+    <Box
+      sx={{
+        height: "100%",
+        pointerEvents: "none",
+        display: "flex",
+        alignItems: "center",
+        fontWeight: "bold",
+        marginRight: 0.5,
+      }}
+    >
+      <Box
+        sx={(theme) => ({
+          height: "30px",
+          display: "flex",
+          alignItems: "center",
+          justifyContent: "center",
+          color: theme.palette.gray[50],
+          backgroundColor: theme.palette.gray[20],
+          borderRadius: "2px",
+          px: 1,
+          py: 0.75,
+        })}
+      >
+        <Typography variant="microText">
+          {isMac ? "Cmd" : "Ctrl"} + P
+        </Typography>
+      </Box>
```

Comment:
Nit: this has a slightly stronger font-weight in the designs.
Currently:
<img width="100" alt="image" src="https://user-images.githubusercontent.com/42802102/158613664-6bb81d8b-2ba3-4bff-8454-b36b183e9151.png">
Should be:
<img width="137" alt="image" src="https://user-images.githubusercontent.com/42802102/158613711-a5296045-3c36-456e-9873-b358f6a61c00.png">

Reply:
Done here: https://github.com/hashintel/hash/pull/397/commits/47b1c62d322a90af5a205ad087d0c1346a103303